src/control/server/config_test.go

```diff
@@ -501,7 +517,89 @@ func TestServer_ConfigDuplicateValues(t *testing.T) {
 
 			conf := NewConfiguration().
 				WithFabricProvider("test").
-				WithGetNetworkDeviceClass(netdetect.GetDeviceClassStub).
+				WithGetNetworkDeviceClass(getDeviceClassStub).
+				WithServers(tc.configA, tc.configB)
+
+			gotErr := conf.Validate(log)
+			CmpErr(t, tc.expErr, gotErr)
+		})
+	}
+}
+
+func TestServer_ConfigNetworkDeviceClass(t *testing.T) {
+	const (
+		ETHER      = 1
+		INFINIBAND = 32
+	)
```

Comment:
IMO these constants should be enumerated in the production code. I think the Go linter will also complain about all-caps consts.

Reply:
Ok, I've added exported constants to the netdetect code.  Do you know why the other constants such as those in the SIG* family (SIGPIPE, SIGKILL .. ) do not fail any lint checks that our system may do?  The ETHER / INFINIBAND constants did not trigger any linter issue that I know of either though.

Reply:
I don't think the linter is actually being run in CI, but there's one that gets installed if you use the Go plugin in VSCode. I think it's golint? That's just particular style convention that I managed to retain from having the tool complain at me in the past. :)